### ISSUE TITLE
Error to convert any currency to bitcoin

### ISSUE THREAD

**Issue by ciciolina** [brpaz/ulauncher-currency#11]
Hi, i found an error when you try to covent any currency to BTC (bitcoin), for example $20 usd to btc, allways show 0.0 BTC, i think its the decimal fractions in the code output, only happens whit BTC, i try to covert usd to other "low" currencies to compare and works perfect the output issue its only with BTC.

Thanks and your extension its fantastic!

**Comment by brpaz** [comment]
Yep, just tested and confirmed the issue. the extension rounds the results to 2 decimal places.
